minerva_analysis/client/src/js/services/dataLayer.js

```diff
@@ -289,6 +289,23 @@ class DataLayer {
         }
     }
 
+    async getAllCells(start_keys) {
+        try {
+            const headers = new Headers();
+            headers.append("Content-Type","application/octet-stream");
+            headers.append("Content-Encoding","gzip");
```

Comment:
Here we expect gzipped binary data. The browser automatically decompresses the bytes, which we then process as Uint32 or Float32.